src/azure-cli/azure/cli/command_modules/vm/_actions.py

```diff
@@ -227,13 +249,23 @@ def _create_image_instance(publisher, offer, sku, version):
     }
 
 
-def _get_latest_image_version(cli_ctx, location, publisher, offer, sku):
-    top_one = _compute_client_factory(cli_ctx).virtual_machine_images.list(location,
-                                                                           publisher,
-                                                                           offer,
-                                                                           sku,
-                                                                           top=1,
-                                                                           orderby='name desc')
-    if not top_one:
-        raise CLIError("Can't resolve the version of '{}:{}:{}'".format(publisher, offer, sku))
+def _get_latest_image_version(cli_ctx, location, publisher, offer, sku, edge_zone=None):
+    if edge_zone is not None:
+        from azure.cli.core.commands.client_factory import get_mgmt_service_client
+        from azure.cli.core.profiles import ResourceType
+        from azure.cli.core.azclierror import InvalidArgumentValueError
```

Comment:
The import of  `InvalidArgumentValueError` needs to be put to the outer layer, because it is also used in the else branch below